src/renderer/intl_provider.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {IntlProvider as BaseIntlProvider} from 'react-intl';
+
+import {GET_LANGUAGE_INFORMATION, RETRIEVED_LANGUAGE_INFORMATION} from 'common/communication';
+
+import {Language} from '../../i18n/i18n';
+
+type State = {
+    language?: Language;
+}
+
+export function t(id: string) {
```

Comment:
Is this still needed with the other version of `t` around?

Reply:
After the changes I made above it won't be, but I wanted to use `t()` purely as a marker function in the renderer process and not in the main. It will change once other changes are made.